recipes/orix/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "orix" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5ede74e8cded3ad055984f15c2bbb989df6805689ca1b73ac6734b83d4147a7b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3
+  run:
+    - python >=3
+    - ipython
+
+test:
+  requires:
+    - pytest
+  imports:
+    - orix
+    - orix.base
+    - orix.grid
+    - orix.io
+    - orix.plot
+    - orix.quaternion
+    - orix.scalar
+    - orix.vector
+  commands:
+    - pytest --pyargs orix
+
+about:
+  home: https://github.com/pyxem/orix
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE
```

Comment:
It seems that the license is not being packaged with this project
You can package it manually then
Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

Reply:
thanks, packaged this.